geometry_msgs/Twist desired_vel
float64[] q_in
---
float64[] q_out
